CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/paths/ShortestPaths2Plugin.java

```diff
@@ -27,6 +29,7 @@
  */
 //@ServiceProvider(service = Plugin.class)
 //@NbBundle.Messages("ShortestPaths2Plugin=Shortest Paths 2")
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```

Comment:
How about "ANALYTIC" here?

Reply:
Done, and switched plugin type to search as it searches the graph